mkosi/__init__.py

```diff
@@ -5631,7 +5632,6 @@ def load_distribution(args: argparse.Namespace) -> argparse.Namespace:
 
 def parse_args(argv: Optional[List[str]] = None) -> Dict[str, argparse.Namespace]:
     """Load default values from files and parse command line arguments
-
```

Comment:
An unrelated whitespace change slipped in here. Can you remove that from the commit?